internal/pkg/service/buffer/store/export.go

```diff
@@ -120,7 +119,7 @@ func (s *Store) updateExportOp(ctx context.Context, oldValue, newValue model.Exp
 		ops = append(ops, closeFileOp, closeSliceOp, createFileOp, createSliceOp)
 	}
 
-	return op.MergeToTxn(ctx, ops...), nil
+	return op.MergeToTxn(ops...), nil
```

Comment:
Tu sa zbytocne musel predavat `ctx`, a realne v kode sa tak zadaval `2x`.
Je to drobnost, ale uz sa mi podarilo omylom zmenit iba jeden `ctx`, tak som to fixol.
Je to v kode na viacerych miestach.

![image](https://user-images.githubusercontent.com/19371734/211218376-398c89d8-8ccb-4372-9ed5-b71504a9835a.png)

Reply:
Ono nedava zmysel, aby si uz pri definicii potreboval `ctx`.